sdk/couchbase-core/tests/orphan_reporter_test.rs

```diff
@@ -0,0 +1,165 @@
+use crate::common::default_agent_options::create_default_options;
+use crate::common::helpers::{generate_bytes_value, generate_key};
+use crate::common::test_config::{setup_test, take_captured_logs, with_log_capture};
+use couchbase_core::agent::Agent;
+use couchbase_core::options::agent::OrphanReporterConfig;
+use couchbase_core::options::crud::UpsertOptions;
+use couchbase_core::orphan_reporter::OrphanReporter;
+use couchbase_core::retryfailfast::FailFastRetryStrategy;
+use std::sync::Arc;
+use std::time::Duration;
+
+mod common;
+
+#[test]
+fn test_orphan_reporter_logs() {
+    setup_test(async |config| {
+        let _guard = with_log_capture();
+
+        let orphan_reporter = OrphanReporter::new(
+            OrphanReporterConfig::default().reporter_interval(Duration::from_secs(1)),
+        );
+        let agent_opts = create_default_options(config)
+            .await
+            .orphan_reporter_handler(orphan_reporter.get_handle());
+
+        let agent = Agent::new(agent_opts).await.unwrap();
+
+        let strat = Arc::new(FailFastRetryStrategy::default());
+        let value = generate_bytes_value(32);
+
+        // Let agent bootstrap
+        tokio::time::sleep(Duration::from_secs(5)).await;
```

Comment:
This shouldn't be necessary should it?

Reply:
Replaced with waituntilready, which was needed on my testing env.